### ISSUE TITLE
quotes in import look weird

### ISSUE THREAD

**Issue by shriram** [racket/racket#2796]
When you import from a module named by a string, you get this:

![image](https://user-images.githubusercontent.com/75030/63687133-c349d980-c7d1-11e9-9b3a-d32f008dbc3d.png)

The double quotes, and of two different kinds of quotes, all looks rather ugly. Since quotes already have a special meaning in that position (and the typographical differences between different kinds of quotes are lost on some people), why not use a different marker for the outer quotes instead, like «ducks feet»?

**Comment by sorawee** [comment]
This is how @shriram's suggestion looks. Not sure if it's better or worse...

<img width="277" alt="Screen Shot 2020-09-02 at 20 57 03" src="https://user-images.githubusercontent.com/9099577/92069689-dbe2bd00-ed5e-11ea-8520-381b9c25f481.png">

**Comment by jackfirth** [comment]
Why not just drop the outer quotes entirely when the inner module is a quoted string? So it would say `imported from "a.rkt"`.

**Comment by rfindler** [comment]
Another thing we could do is add styling information so the `"a.rkt"` would be in TT font. (We would probably want to loop in @greghendershott to get the best tradeoff between backwards compatibility and making things work with Emacs mode if we go this route.)

**Comment by greghendershott** [comment]
I agree it looks weird.

Personally, I think the best fix is simply to remove the fancy quotes.

(So that whatever follows `"imported by "` is simply the text you'd see in Racket source. Although I _usually_ ~dis~like **fancy** `formatting`  :smile:, I think the context makes it clear that the text must be a module name, even if it is plain text.)

---

When I encountered this, I decided just to "cleanse" it on my, consuming end:

```racket
    (define/override (syncheck:add-mouse-over-status _text beg end status)
      (when (valid-beg/end? beg end)
        ;; Avoid silly "imported from “\"file.rkt\"”"
        (define cleansed (regexp-replace* #px"[“””]" status ""))
        (interval-map-update*! im-mouse-overs beg end
                               (λ (s) (set-add s cleansed))
                               (set cleansed))
```

I probably should have filed an issue here, to ask about the intent. But I think at the time I was juggling a dozen things getting this working, and forgot.

Anyway, this "cleansing" would work fine if you simply omit the fancy quotes. It would continue to help with older versions of `drracket/check-syntax`, and simply be harmless and N/A for newer versions.
